### ISSUE TITLE
`kalite manage setup` erroneously warns about installing as root user

### ISSUE THREAD

**Issue by indirectlylit** [learningequality/ka-lite#5032]
## Summary

Following the [getting started instructions](https://github.com/learningequality/ka-lite/wiki/Getting-started), running

    kalite manage setup --username=admin --password=pass --noinput

prints:

```
-------------------------------------------------------------------
WARNING: You are installing KA-Lite as root user!
	Installing as root may cause some permission problems while running
	as a normal user in the future.
-------------------------------------------------------------------
```

However, this is not true.

## Branch or installer method

`develop` branch, OSX 10.9.5, Python 2.7.11

## How to reproduce

Check out a fresh copy of the `develop`, create a new virtualenv environment, and follow the [instructions](https://github.com/learningequality/ka-lite/wiki/Getting-started) until step 8.

This seems to stem from `setup.py`, and was already noted in the code:

```python
        # Tried not to be os-specific, but ... hey. :-/
        # benjaoming: This doesn't work, why is 502 hard coded!? Root is normally
        # '0' And let's not care about stuff like this, people can be free to
        # run this as root if they want :)
        if not is_windows() and hasattr(os, "getuid") and os.getuid() == 502:
            print(
                "-------------------------------------------------------------------")
            print("WARNING: You are installing KA-Lite as root user!")
            print(
                "\tInstalling as root may cause some permission problems while running")
            print("\tas a normal user in the future.")
            print(
                "-------------------------------------------------------------------")
            if options["interactive"]:
                if not raw_input_yn("Do you wish to continue and install it as root?"):
                    raise CommandError("Aborting script.\n")
```

**Comment by radinamatic** [comment]
Could you please edit your issue  - fill in the rest of the issue template and delete non applicable parts - so we can better understand the actual problem? Thanks! :smiley:

**Comment by indirectlylit** [comment]
slipped and submitted before complete. updated now.

**Comment by benjaoming** [comment]
Nicely spotted, @indirectlylit.. I always wondered why someone had put 502 there.

It doesn't make sense.

On a Mac OSX, the first user is 501 and the second user is 502. Someone must have wrongly assumed that this would be the root user on *any* system!?

We should use a check like `os.geteuid() == 0`.

**Comment by benjaoming** [comment]
Fixed in #5222
